ros_gz_shims/ros_ign_image/src/image_bridge_shim.cpp

```diff
@@ -0,0 +1,30 @@
+// Shim to redirect "ros_ign_image image_bridge" call to "ros_gz_image image_bridge"
+
+#include <string>
```

Comment:
Copyright and include

Reply:
https://github.com/gazebosim/ros_gz/pull/281/commits/c79ba8bc3ccc36eba15084a1c378a3f08e4fffa4 :>